recipes/libtinyspline/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "libtinyspline" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/msteinbeck/tinyspline/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3ea31b610dd279266f26fd7ad5b5fca7a20c0bbe05c7c32430ed6aa54d57097a
+
+build:
+  script: 
+    - mkdir -p build
+    - cd build
+    - cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DTINYSPLINE_BUILD_EXAMPLES=OFF -DTINYSPLINE_BUILD_DOCS=OFF -DTINYSPLINE_INSTALL_LIBRARY_DIR=lib -DCMAKE_INSTALL_PREFIX=${PREFIX}  # [unix]
+    - cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DTINYSPLINE_BUILD_EXAMPLES=OFF -DTINYSPLINE_BUILD_DOCS=OFF -DTINYSPLINE_INSTALL_LIBRARY_DIR=lib -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%  # [win]
+    - cmake --build .
+    - strip --strip-unneeded lib/libtinyspline*  # [linux]
+    - cmake --build . --target install
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libtinyspline', max_pin='x') }}
+
+requirements:
+  build:
+    - cmake
+    - binutils  # [linux]
```

Comment:
This should not be required.
```suggestion
```

Reply:
I added `binutils` to have the  command `strip`. Otherwise, the build was failing with "command not found". Also, stripping only works on Linux (and all binaries should be stripped).

Reply:
There should be a `$STRIP` command set that comes in via the compiler. Please use that one.

Reply:
In the end, I switched to using `cmake --install . --strip` instead of manually using the command `$STRIP` because cmake knows how to do it across all platforms.